.solhint.json

```diff
@@ -3,12 +3,16 @@
     "rules": {
       "indent": ["error", 4],
       "compiler-fixed": "off",
-      "max-states-count": ["warn",25],
+      "max-states-count": ["warn", 25],
       "no-simple-event-func-name": "off",
       "two-lines-top-level-separator": "off",
       "mark-callable-contracts": "off",
-      "reason-string": "on",
-      "compiler-version": ["error","^0.8.6"],
-      "func-visibility": ["error",{"ignoreConstructors": true}]
+      "reason-string": ["error", {"maxLength": 64}],
+      "compiler-version": ["error", "^0.8.6"],
+      "func-visibility": ["error", {"ignoreConstructors": true}],
+      "max-line-length": ["warn", 179],
```

Comment:
made it too long for now, need to change to 79 later when we lint everything

Reply:
or i can do it now in this pr

Reply:
please do not change the linter rules :)